@navikt/core/react/src/theme/Theme.tsx

```diff
@@ -73,11 +73,12 @@ const [ThemeProvider, useThemeInternal] = createContext<ThemeContext>({
 export type ThemeProps = {
   className?: string;
   /**
-   * Sets default background when enabled
+   * Whether to apply the default background.
+   * @default true
```

Comment:
We should probably mention that only the first/root-level Theme applies background by default